### DISCUSSION TITLE
How to get wifi password

### DISCUSSION THREAD

**Discussion by MilkMaker23** [3kh0/ext-remover#791] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE]
Hello i am trying to get my school wifi password i used caub it used to work but now it doesnt also chrome://netexport is blocked

**Comment by S-PScripts** [comment]
Did you mean **chrome://net-export/**

https://github.com/S-PScripts/chromebook-utilities/blob/main/Exploits/WIFI%20Password

**Reply by MilkMaker23** [reply]
yeah

**Reply by liamb410** [reply]
did u figure it out

**Reply by S-PScripts** [reply]
don't necropost

**Comment by 88411** [comment]
Follow the instructions : https://luphoria.com/netlog-policy-password-tool

**Reply by 88411** [reply]
If you can't access the link try this: https://github.com/3kh0/ext-remover/discussions/1163

**Reply by S-PScripts** [reply]
issue dated (old), someone necroposted.

**Comment by Moose626262-plane** [comment]
Thank You
